lte/gateway/python/scripts/generate_oai_config.py

```diff
@@ -189,6 +189,11 @@ def _get_restricted_plmns(service_mconfig):
         return service_mconfig.restricted_plmns
     return {}
 
+def _get_restricted_imeis(service_mconfig):
```

Comment:
this lint recommendation is legit, there should be 2 spaces separating non-member functions